browser/src/control/Control.NotebookbarImpress.js

```diff
@@ -137,140 +153,138 @@ L.Control.NotebookbarImpress = L.Control.NotebookbarWriter.extend({
 	},
 
 	getFileTab: function() {
-		var hasRevisionHistory = L.Params.revHistoryEnabled;
-		var hasPrint = !this._map['wopi'].HidePrintOption;
-		var hasRepair = !this._map['wopi'].HideRepairOption;
-		var hasSaveAs = !this._map['wopi'].UserCanNotWriteRelative;
-		var hideDownload = this._map['wopi'].HideExportOption;
-		var hasShare = this._map['wopi'].EnableShare;
-		var hasGroupedDownloadAs = !!window.groupDownloadAsForNb;
-		var hasGroupedSaveAs = window.uiDefaults && window.uiDefaults.saveAsMode === 'group';
-		var hasRunMacro = !(window.enableMacrosExecution  === 'false');
-		var hasSave = !this._map['wopi'].HideSaveOption;
-		var content = [];
-
-		if (hasSave) {
-			content.push({
+		var content = [
+			!this._map['wopi'].HideSaveOption ?
+			{
 				'type': 'toolbox',
 				'children': [
 					{
 						'id': 'file-save',
 						'type': 'bigtoolitem',
 						'text': _('Save'),
-						'command': '.uno:Save'
+						'command': '.uno:Save',
+						'accessibility': { focusBack: true, combination: 'SV', de: null }
 					}
 				]
-			});
-		}
-
-		if (hasSaveAs) {
-			if (hasGroupedSaveAs) {
-				content.push({
-					'id': 'saveas',
+			}: {},
+			(!this._map['wopi'].UserCanNotWriteRelative) ?
+			(
+				(window.uiDefaults && window.uiDefaults.saveAsMode === 'group') ?
+				{
+					'id': 'file-saveas',
```

Comment:
the button is coming out without icon please add icon alias:
```diff
diff --git a/browser/src/control/Control.JSDialogBuilder.js b/browser/src/control/Control.JSDialogBuilder.js
index 6abb6e6627..d45d330458 100644
--- a/browser/src/control/Control.JSDialogBuilder.js
+++ b/browser/src/control/Control.JSDialogBuilder.js
@@ -2320,6 +2320,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		}
 
 		var iconURLAliases = {
+			'file-saveas': 'saveas',
 			'home-search': 'recsearch',
 			'addmb-menu': 'ok',
 			'closetablet': 'view',
```

Also the save as should be a dropdown but it looks like this now:
![image](https://github.com/CollaboraOnline/online/assets/65948705/29865db7-b54d-4b87-88d2-2153d439785b)